src/instructlab/cli/model/train.py

```diff
@@ -467,6 +467,7 @@ def train(
                 eval_gpus=ctx.obj.config.evaluate.gpus,
                 training_journal=training_journal,
             )
+            click.echo("✅ Model train with accelerated completed successfully!")
```

Comment:
These are all tricky since other messages are being printed throughout the code.  For example with case I think you might already get a:

Training finished! Best final checkpoint: /tmp/tmp.01L0NjBZlG/.local/share/instructlab/skills-only/phase2/checkpoints/hf_format/samples_616 with score: 8.368421052631579

Also nit:  This message might be better worded as:

Accelerated model training completed successfully!

(Similar for the other train messages below)

Reply:
yeah, just checked, seems `finished` only for accelerated train, there is no msg for `simple` and `full`